src/alitra/models/bounds.py

```diff
@@ -0,0 +1,40 @@
+from dataclasses import dataclass
+
+from alitra.frame_dataclasses import Point
+
+
+# Point 1 and 2 should be on the diagonal corners of the bounding cube
+@dataclass
+class Bounds:
+    point1: Point
+    point2: Point
+
+    def __post_init__(self):
+        if self.point1.frame == self.point2.frame:
+            self.frame = self.point1.frame
+        else:
+            raise FrameException("The frames of the bounding points are not the same")
+        points: list = [self.point1, self.point2]
+        self.x_max = max(point.x for point in points)
```

Comment:
Some questions for clarification: 
a) Is rectangular bounds sufficient? - presumably this may cause some maps to overlap?. 
b) What is the purpose of this check? (given the above limitation).